utils/address.go

```diff
@@ -17,7 +17,7 @@ func ParseCLIAddress(clientCtx client.Context, address string) (string, error) {
 		// empty address --> address = creator
 		address = clientCtx.GetFromAddress().String()
 	} else {
-		if IsBech32Address(address) {
+		if IsBech32Address(address) || address == "empty_provider" {
```

Comment:
thanks for the finding, can you use the const instead of hard coded string?

Reply:
Thx for the replay. I can't import const from dualstaking module because of cycle imports. I have created new const